src/main/java/seedu/address/logic/parser/MarkCommandParser.java

```diff
@@ -0,0 +1,41 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_WEEK;
+
+import java.util.stream.Stream;
+
+import seedu.address.logic.commands.MarkCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.person.Name;
+
+/**
+ * Parses input arguments and creates a new MarkCommand object
+ */
+public class MarkCommandParser implements Parser<MarkCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the seedu.address.logic.commands.MarkCommand
+     * and returns a seedu.address.logic.commands.MarkCommand object for execution.
```

Comment:
I think there needs to be an empty line between the description and the parameters. Refer to the [coding standard](https://se-education.org/guides/conventions/java/intermediate.html#comments) for more info.